container/apptabs.go

```diff
@@ -288,16 +288,18 @@ func (r *appTabsRenderer) Refresh() {
 }
 
 func (r *appTabsRenderer) buildOverflowTabsButton() (overflow *widget.Button) {
-	overflow = widget.NewButtonWithIcon("", moreIcon(r.appTabs), func() {
+	overflow = &widget.Button{Icon: moreIcon(r.appTabs), Importance: widget.LowImportance, OnTapped: func() {
 		// Show pop up containing all tabs which did not fit in the tab bar
 
-		var items []*fyne.MenuItem
-		for i := len(r.bar.Objects[0].(*fyne.Container).Objects); i < len(r.appTabs.Items); i++ {
+		itemLen, objLen := len(r.appTabs.Items), len(r.bar.Objects[0].(*fyne.Container).Objects)
+		items := make([]*fyne.MenuItem, 0, itemLen-objLen)
+		for i := objLen; i < itemLen; i++ {
 			item := r.appTabs.Items[i]
```

Comment:
This is optional but I think now we can avoid creating this variable as we now capture the index?
```go
items = append(items, fyne.NewMenuItem(r.appTabs.Items[index].Text, func() {
	r.appTabs.SelectIndex(index)
	if r.appTabs.popUpMenu != nil {
		r.appTabs.popUpMenu.Hide()
		r.appTabs.popUpMenu = nil
	}
}))
```

Reply:
Good point. Will fix that tomorrow if I have the time 🙂

Reply:
Fixed now.